network-upgrades/mainnet-upgrades/paris.md

```diff
@@ -39,7 +39,7 @@ Once the proof-of-work to proof-of-stake transition has completed, an additional
 | Ropsten | N/A | N/A | N/A | 
 | Sepolia | 1735371 | August 17, 2022 | `0xb96cbd13` | 
 | Goerli  | TBD | TBD | TBD | 
-| Mainnet | TBD | TBD | TBD |  
+| Mainnet | 15537394 | September 15, 2022 | TBD |  
```

Comment:
This is a table of `FORK_NEXT` blocks, but the number you have put in it is block number of the first POS block. The entry should remain "TBD" since we haven't decided when mainnet `FORK_NEXT` will happen.

Reply:
+1